bootcfg/storage/storagepb/storage.proto

```diff
@@ -25,8 +25,10 @@ message Profile {
   string ignition_id = 3;
   // cloud config id
   string cloud_id = 4;
+  // generic config id
+  string generic_id = 5;
   // support network boot / PXE
-  NetBoot boot = 5;
+  NetBoot boot = 6;
```

Comment:
Ah, thanks for the reference. Makes sense in retrospect :) My base instinct was to group the id's.